configs/nextflow.hopper.config

```diff
@@ -0,0 +1,170 @@
+	}
+	withName: 'bwa_mem|bwa_index' {
+		container = "/fs1/pipelines/JASEN/container/bwa.sif"
+		cpus = 16
+		memory = '32 GB'
+		time = '1h'
+		publishDir = "/fs1/results_dev/jasen/bwa"
+	}
+	withName: 'samtools_sort|samtools_index' {
+		container = "/fs1/pipelines/JASEN/container/samtools.sif"
+		publishDir = "/fs1/results_dev/jasen/samtools"
+	}
+	withName: spades {
+		container = "/fs1/pipelines/JASEN/container/spades.sif"
+		cpus = 16
+		memory = '64 GB'
+		time = '2h'
+		publishDir = "/fs1/results_dev/jasen/spades"
+	}
+	withName: quast {
+		container = "/fs1/pipelines/JASEN/container/quast.sif"
+		publishDir = "/fs1/results_dev/jasen/quast"
+	}
+	withName: mlst {
+		container = "/fs1/pipelines/JASEN/container/mlst.sif"
+		publishDir = "/fs1/results_dev/jasen/mlst"
+	}
+	withName: kraken {
+		cpus = 16
+		memory = '48 GB'
+		time = '2h'
+		container = "/fs1/pipelines/JASEN/container/kraken2.sif"
+		publishDir = "/fs1/results_dev/jasen/kraken"
+	}
+	withName: bracken {
+		cpus = 16
+		memory = '48 GB'
+		time = '2h'
+		container = "/fs1/pipelines/JASEN/container/bracken.sif"
+		publishDir = "/fs1/results_dev/jasen/kraken"
+	}
+	withName: resfinder {
+		container = "/fs1/pipelines/JASEN/container/resfinder.sif"
+		publishDir = "/fs1/results_dev/jasen/resfinder"
+	}
+	withName: virulencefinder {
+		container = "/fs1/pipelines/JASEN/container/virulencefinder.sif"
+		publishDir = "/fs1/results_dev/jasen/virulencefinder"
+	}
+	withName: freebayes {
+		container = "/fs1/pipelines/JASEN/container/freebayes.sif"
+		publishDir = "/fs1/results_dev/jasen/freebayes"
+	}
+	withName: 'chewbbaca_allelecall|chewbbaca_split_results|chewbbaca_split_missing_loci|chewbbaca_create_batch_list' {
+		//executor='local'
+		container = "/fs1/pipelines/JASEN/container/chewbbaca_3.1.0.sif"
+		cpus = 7
+		publishDir = "/fs1/results_dev/jasen/chewbbaca"
+	}
+	withName: ariba_summary_to_json {
+		container = "/fs1/pipelines/JASEN/container/perl_json_3A4.02.sif"
+		publishDir = "/fs1/results_dev/jasen/ariba"
+	}
+	withName: post_align_qc {
+		container = "/fs1/pipelines/JASEN/container/postAlignQc.sif"
+		publishDir = "/fs1/results_dev/jasen/postalignqc"
+	}
+	withName: export_to_cdm {
+		publishDir = "/fs1/results_dev/cron/qc"
+	}
+	withName: create_analysis_result {
+		container = "/fs1/pipelines/JASEN/container/pythonScripts.sif"
+		publishDir = "/fs1/results_dev/jasen/analysis_result"
+	}
+	withName: save_analysis_metadata {
+		publishDir = "/fs1/results_dev/jasen/analysis_metadata"
+	}
+	withName: 'mask_polymorph_assembly' {
+		publishDir = "/fs1/results_dev/jasen/misc"
+	}
+}
+
+params {
+	root = "/fs1/pipelines/JASEN"
+	// DATABASES //
+	resfinderDb = "${root}/assets/resfinder_db"
+	pointfinderDb = "${root}/assets/pointfinder_db"
+	virulencefinderDb = "${root}/assets/virulencefinder_db"
+	mlstBlastDb = "${root}/assets/blast/mlst.fa"
+	//krakenDb = "/fs1/resources/ref/micro/krakenstd"
+	// INPUT & OUTPUT //
+	localTempDir = "/tmp"
```

Comment:
Never use /tmp for anything

Reply:
Good to know. Leave it as an empty string?

Reply:
Depends on what you are planning. There is a tempdir on every local node
jonas@rs-n18: ~$ echo $TMPDIR
/local/scratch
jonas@rs-n18: ~$